QUEST_LV_0300.txt

```diff
@@ -1,42 +1,42 @@
-﻿QUEST_LV_0300_20150323_000001
-QUEST_LV_0300_20150323_000002
-QUEST_LV_0300_20150323_000003
-QUEST_LV_0300_20150323_000004
-QUEST_LV_0300_20150323_000005
-QUEST_LV_0300_20150323_000006
-QUEST_LV_0300_20150323_000007
-QUEST_LV_0300_20150323_000008
-QUEST_LV_0300_20150323_000009
-QUEST_LV_0300_20150323_000010
-QUEST_LV_0300_20150323_000011
-QUEST_LV_0300_20150717_000012
-QUEST_LV_0300_20150717_000013
-QUEST_LV_0300_20150717_000014
-QUEST_LV_0300_20150717_000015
-QUEST_LV_0300_20150717_000016
-QUEST_LV_0300_20150717_000017
-QUEST_LV_0300_20150717_000018
-QUEST_LV_0300_20150717_000019
-QUEST_LV_0300_20150717_000020
-QUEST_LV_0300_20150717_000021
-QUEST_LV_0300_20150717_000022
-QUEST_LV_0300_20150717_000023
-QUEST_LV_0300_20150717_000024
-QUEST_LV_0300_20150717_000025
-QUEST_LV_0300_20150717_000026
-QUEST_LV_0300_20150717_000027
-QUEST_LV_0300_20150717_000028
-QUEST_LV_0300_20150717_000029
-QUEST_LV_0300_20150717_000030
-QUEST_LV_0300_20150717_000031
-QUEST_LV_0300_20150717_000032
-QUEST_LV_0300_20150717_000033
-QUEST_LV_0300_20150717_000034
-QUEST_LV_0300_20150717_000035
-QUEST_LV_0300_20150717_000036
-QUEST_LV_0300_20150717_000037
-QUEST_LV_0300_20150717_000038
-QUEST_LV_0300_20150717_000039
+QUEST_LV_0300_20150323_000001	$To support the Knights of Kaliss, we were deployed here from headquarters.{nl} If you want to know more, head over to our headquarters at Verkti Square.
+QUEST_LV_0300_20150323_000002	$I've been a merchant all my life. When Medzio Diena hit, I thought of setting up shop here as I heard it was safest here.
+QUEST_LV_0300_20150323_000003	$I met many wandering Revelators around here. I really think they were...different, and it wasn't just that they were strong.
```

Comment:
I believe think should be changed to thought in this context. May be wrong, does the original korean have it in its present tense?